app/src/main/java/com/epmedu/animeal/AnimealApplication.kt

```diff
@@ -1,7 +1,27 @@
 package com.epmedu.animeal
 
 import android.app.Application
+import android.util.Log
+import com.amplifyframework.AmplifyException
+import com.amplifyframework.core.Amplify
 import dagger.hilt.android.HiltAndroidApp
 
 @HiltAndroidApp
-class AnimealApplication : Application()
\ No newline at end of file
+class AnimealApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        configureAmplify()
+
+    }
+
+    fun configureAmplify() {
+        try {
+            Amplify.configure(applicationContext)
+            Log.i("MyAmplifyApp", "Initialized Amplify")
```

Comment:
Could you define AnimealApplication tag and refer to it at logs. something like this:
`private companion object {
    private const val TAG = "AnimealApplication"
}`

Reply:
I would prefer not going with the default logging-apis.
In my opinion we should start using https://github.com/JakeWharton/timber or some analogue & this line should be nuked.

Prepping the logging-lib seemed out of scope of this task.
Not sure, but the `Amplify#cofigure` will most likely also move out of here, with the boilerplate setup of hilt-dependency definitions.